foodnet/membership/urls.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+from django.conf import settings
+from django.conf.urls import patterns, url
+
+all_urls = [
+    'foodnet.membership.views',
+    url(r'profile/$',
+        'profile',
+        name="profile"),
+]
+
+urlpatterns = patterns(*all_urls)
```

Comment:
The whole `patterns` thing is being deprecated, and a much simpler list based system is now how one should do URLConfs. Do this instead:
```
from django.conf.urls import url
from .views import profile_view

urlpatterns = [
    url(r'profile/$', profile_view, name="profile"),
]
```

See https://docs.djangoproject.com/en/1.8/topics/http/urls/ for the new syntax :)